pkg/printers/internalversion/describe.go

```diff
@@ -2764,6 +2764,12 @@ func describeStatefulSet(ps *apps.StatefulSet, selector labels.Selector, events
 		printLabelsMultiline(w, "Labels", ps.Labels)
 		printAnnotationsMultiline(w, "Annotations", ps.Annotations)
 		w.Write(LEVEL_0, "Replicas:\t%d desired | %d total\n", ps.Spec.Replicas, ps.Status.Replicas)
+		w.Write(LEVEL_0, "UpdateStrategyType:\t%s\n", ps.Spec.UpdateStrategy.Type)
+		if ps.Spec.UpdateStrategy.RollingUpdate != nil {
+			ru := ps.Spec.UpdateStrategy.RollingUpdate
+			w.Write(LEVEL_0, "RollingUpdateStrategy:\t%d partition\n", ru.Partition)
```

Comment:
This can be `LEVEL_1`, `"Partition:"`. 

Suggest not printing partition when the value is 0 (no partition).